docs/tables/aws_wellarchitected_answer.md

```diff
@@ -0,0 +1,131 @@
+# Table: aws_wellarchitected_answer
+
+The answers of a lens review in a Well-Architected workload
+
+**Important notes:**
+
+- For improved performance, it is advised that you use the optional qual `workload_id`, `pillar_id` and `lens_alias` to limit the result set to a specific workload, lens or pillar.
+
+## Examples
+
+### Basic info
+
+```sql
+select
+  a.question_id,
+  a.lens_alias,
+  a.workload_id,
+  a.is_applicable,
+  a.pillar_id,
+  a.question_title,
+  a.risk,
+  a.reason,
+  a.region
+from
+  aws_wellarchitected_answer a;
+```
+
+### Get number of questions per piller
+
+```sql
+select
+  a.workload_id,
+  a.pillar_id,
+  count(a.question_id) as total_questions
+from
+  aws_wellarchitected_answer a
+group by
+  a.workload_id,
+  a.pillar_id;
+```
+
+### List all the questions along with the choices for a workload
```

Comment:
The description does not match the example query. Could you please update the description/query?